projects/angular/src/data/datagrid/_datagrid.clarity.scss

```diff
@@ -1508,3 +1503,8 @@
 
   // END Calculation classes.
 }
+
+.datagrid-numeric-filter-input-wrapper {
```

Comment:
Shouldn't this be scoped at least inside the `@include mixins.exports('datagrid.clarity') {}` block? I think this should be moved the same location with the file that the removed `.datagrid-filter-input-spacer` styles were.

Reply:
Make sense to me